source/fundamentals/indexes.txt

```diff
@@ -242,9 +244,20 @@ language as an option when creating the index.
    Text indexes differ from the more powerful :atlas:`Atlas full text search indexes </atlas-search/>` Atlas users
    should use Atlas search.
 
-.. warning::
+.. warning:: Restriction on one text index per collection. 
+
+   If you want to create a text index for multiple text fields, you need
+   to create an compound index to include them. 
```

Comment:
I wasn't able to find any information in either this page or the Text Search one as to whether a text search performs a search on all the indexed fields. This might not be easy enough for someone to assume, so it could be useful to make that information explicit.